deeppavlov/requirements/faiss.txt

```diff
@@ -0,0 +1 @@
+faiss==1.6.3
```

Comment:
There is no faiss 1.6.3 in PyPI. Eather you mean faiss-cpu==1.6.3, or faiss==1.5.3